src/EnergyPlus/OutputReportTabular.cc

```diff
@@ -4767,6 +4804,263 @@ namespace OutputReportTabular {
+        using DataHeatBalance::NumRefrigeratedRacks;
+        using DataHeatBalance::SysTotalHVACRejectHeatLoss;
+        using DataHeatBalance::SysTotalHVACReliefHeatLoss;
+        using DataHeatBalance::ZoneTotalExfiltrationHeatLoss;
+        using DataHeatBalance::ZoneTotalExhaustHeatLoss;
+        using DataHeatBalSurface::SumSurfaceHeatEmission;
+        using DataHVACGlobals::AirCooled;
+        using DataHVACGlobals::EvapCooled;
+        using DataHVACGlobals::WaterCooled;
+        using DXCoils::DXCoil;
+        using DXCoils::NumDXCoils;
+        using EvaporativeCoolers::EvapCond;
+        using EvaporativeCoolers::NumEvapCool;
+        using EvaporativeFluidCoolers::NumSimpleEvapFluidCoolers;
+        using EvaporativeFluidCoolers::SimpleEvapFluidCoolerReport;
+        using FluidCoolers::NumSimpleFluidCoolers;
+        using FluidCoolers::SimpleFluidCoolerReport;
+        using HeatingCoils::HeatingCoil;
+        using HeatingCoils::NumHeatingCoils;
+        using HVACVariableRefrigerantFlow::NumVRFCond;
+        using HVACVariableRefrigerantFlow::VRF;
+        using MixedAir::NumOAControllers;
+        using MixedAir::OAController;
+        using PackagedThermalStorageCoil::NumTESCoils;
+        using PackagedThermalStorageCoil::TESCoil;
+        using PlantChillers::ConstCOPChiller;
+        using PlantChillers::ConstCOPChillerReport;
+        using PlantChillers::ElectricChiller;
+        using PlantChillers::ElectricChillerReport;
+        using PlantChillers::EngineDrivenChiller;
+        using PlantChillers::EngineDrivenChillerReport;
+        using PlantChillers::GTChiller;
+        using PlantChillers::GTChillerReport;
+        using PlantChillers::NumConstCOPChillers;
+        using PlantChillers::NumElectricChillers;
+        using PlantChillers::NumEngineDrivenChillers;
+        using PlantChillers::NumGTChillers;
+        using RefrigeratedCase::RefrigRack;
+        using VariableSpeedCoils::NumVarSpeedCoils;
+        using VariableSpeedCoils::VarSpeedCoil;
+        using WaterThermalTanks::AmbientTempOutsideAir;
+        using WaterThermalTanks::NumWaterThermalTank;
+        using WaterThermalTanks::WaterThermalTank;
+
+        static int iOACtrl(0);
+        static int iCoil(0);
+        static int iCooler(0);
+        static int iChiller(0);
+        static int iBoiler(0);
+        static int iTank(0);
+        static int iRef(0);
+
+        static Real64 H2OHtOfVap_HVAC = Psychrometrics::PsyHgAirFnWTdb(DataEnvironment::OutHumRat, DataEnvironment::OutDryBulbTemp);
+        static Real64 RhoWater = Psychrometrics::RhoH2O(DataEnvironment::OutDryBulbTemp);
+
+        SysTotalHVACReliefHeatLoss = 0;
+        SysTotalHVACRejectHeatLoss = 0;
+
+        if (!DoWeathSim) return;
+
+        if (!displayHeatEmissionsSummary) return; // don't gather data if report isn't requested
+
+        if (IndexTypeKey != stepTypeZone) return; // only add values over the zone timestep basis
```

Comment:
No, this is incorrect. Nearly all of the variables gathered below are at the system timestep, so this needs to accumulate at every HVAC timestep. It think only `SumSurfaceHeatEmission` is at the zone timestep, everything else is HVAC or a mix - see more comments below. So, instead of returning for `!stepTypeZone` this should skip only `BuildingPreDefRep.emiEnvelopConv`.